lib/xeroizer/models/overpayment.rb

```diff
@@ -0,0 +1,51 @@
+module Xeroizer
```

Comment:
Can you add specs for this?

https://developer.xero.com/documentation/api/overpayments/ has a fair amount of detail on what is valid and what isn't